### PR TITLE
Add consumption based emissions

### PR DESCRIPTION
**Pull request by ReemaMohanty**
The corresponding file deals with consumption based emission with reference to our world in data ,updated energy emissions and Ricardo Energy and Environment. #525

### CODE REVIEW

File: openfisca_uk/parameters/consumption/carbon/consumption.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+description: Consumption emissions, [more description]
```
Comment (nikhilwoodruff) [inline openfisca_uk/parameters/consumption/carbon/consumption.yaml]: Could you replace [more description] with something descriptive about this data?

File: openfisca_uk/parameters/consumption/carbon/consumption.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+values:
+  2016-01-01: 543_500_000
+  2017-01-01: 100
```
Comment (nikhilwoodruff) [inline openfisca_uk/parameters/consumption/carbon/consumption.yaml]: To do: add remaining years

**Review by nikhilwoodruff — CHANGES_REQUESTED** [review]

File: openfisca_uk/parameters/consumption/carbon/consumption.yaml (hunk 1 above)
Comment (ReemaMohanty) [reply openfisca_uk/parameters/consumption/carbon/consumption.yaml]: Done. Pls check

Comment (nikhilwoodruff) [reply openfisca_uk/parameters/consumption/carbon/consumption.yaml]: I see it now says `household`, but how about: "Total household consumption-based carbon emissions"?

File: openfisca_uk/parameters/consumption/carbon/consumption.yaml (hunk 2 above)
Comment (nikhilwoodruff) [reply openfisca_uk/parameters/consumption/carbon/consumption.yaml]: Thanks @ReemaMohanty - Though I think we can remove 2020's entry for now as OWID says it's just 2019's entry (and not sure if it's correct - 300?)

File: openfisca_uk/parameters/consumption/carbon/consumption.yaml (hunk 1 above)
Comment (ReemaMohanty) [reply openfisca_uk/parameters/consumption/carbon/consumption.yaml]: Okay ,can we change it to [Carbon emissions] ,[household based consumption]?
I have changed it. Pls check.

Comment (nikhilwoodruff) [reply openfisca_uk/parameters/consumption/carbon/consumption.yaml]: Not sure why we need the square brackets? I think my suggestion is more in line with other parameter descriptions if that's OK.

Comment (ReemaMohanty) [reply openfisca_uk/parameters/consumption/carbon/consumption.yaml]: Square brackets are for  explanatory purpose. Not adding it though.

File: openfisca_uk/parameters/consumption/carbon/consumption.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-description: Carbon emissions, [houshold based consumption]
+description: Carbon emissions, houshold based consumption
```
Comment (MaxGhenis) [inline openfisca_uk/parameters/consumption/carbon/consumption.yaml]:
```suggestion
description: Carbon emissions, household based consumption
```

**Comment by nikhilwoodruff** [comment]
Thanks @ReemaMohanty - looks good on #531 so I'm going to merge this.

**Pull request merged**